Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -536,6 +514,42 @@ defineSuite([
         });
     });
 
+    it('replacement refinement - refines to visible ready children', function() {
+        viewRootOnly();
+        return Cesium3DTilesTester.loadTileset(scene, tilesetUrl).then(function(tileset) {
```

Comment:
Run this test twice, once with `refineToVisible` set to `true` and once with it set to `false`?